vignettes/intro.Rmd

```diff
@@ -0,0 +1,162 @@
+---
+title: "A Brief Introduction to RNeXML"
+author:
+  - Carl Boettiger
+  - Scott Chamberlain
+  - Rutger Vos
+  - Hilmar Lapp
+
+output: html_vignette
+---
+
+<!--
+%\VignetteEngine{knitr::rmarkdown}
+%\VignetteIndexEntry{A Brief Introduction to RNeXMLL}
+-->
+
+
+```{r echo=FALSE}
+library(RNeXML)
+```
+
+Read in a `nexml` file into the `ape::phylo` format:
+
+```{r}
+f <- system.file("examples", "comp_analysis.xml", package="RNeXML")
+nexml <- nexml_read(f)
+tr <- get_trees(nexml) # or: as(nexml, "phylo")
+plot(tr)
+```
+
+Write an `ape::phylo` tree into the `nexml` format:
+
+```{r}
+data(bird.orders)
+nexml_write(bird.orders, "test.xml")
+```
+
+A key feature of NeXML is the ability to formally validate the construction of the data file against the standard (the lack of such a feature in nexus files had lead to inconsistencies across different software platforms, and some files that cannot be read at all).  While it is difficult to make an invalid NeXML file from `RNeXML`, it never hurts to validate just to be sure:
+
+```{r}
+nexml_validate("test.xml")
+```
+
+
+
+Extract metadata from the NeXML file: 
+
+```{r } 
+birds <- nexml_read("test.xml")
+get_taxa(birds)
+get_metadata(birds) 
```

Comment:
I suppose it's fine not to say anything about the `simplify = {TRUE | FALSE}` option here?